pubspec.yaml

```
@@ -8,7 +8,7 @@ environment:
   sdk: ">=2.19.0 <4.0.0"
 
 dependencies:
-  web: ^0.4.0
+  web: ^0.5.1
```

```suggestion
  web: ^0.5.0
```

To be more inclusive, changing to `^0.5.0`.